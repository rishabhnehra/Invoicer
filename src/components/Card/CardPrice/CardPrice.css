.CardPrice {
    border-radius: 10px;
    padding: 16px;
    color: #FFF;
    width: 230px;
    background-color: #9E579D;
    box-shadow: 0px 0px 6px 3px rgba(0,0,0,0.1);
}

.CardPrice .headerSection {
    display: flex;
    justify-content: space-between;
}

.CardPrice .priceSection {
    margin-top: 24px;
    display: flex;
    justify-content: space-between;
}

.CardPrice .headerSection .description h1 {
    margin: 0;
    font-size: 16px;
    font-weight: 600;
}

.CardPrice .headerSection .description p {
    margin: 0;
    font-size: 11px;
    font-weight: 100;
}

.CardPrice .priceSection .quantity {
    font-size: 10px;
    margin: 4px 0 0 0;
    vertical-align: text-bottom;
}

.CardPrice .priceSection .price {
    font-size: 16px;
    margin: 0;
}