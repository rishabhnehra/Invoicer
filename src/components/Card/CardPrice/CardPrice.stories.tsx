import {Story, Meta} from '@storybook/react/types-6-0';

import {CardPrice, CardPriceProps} from './CardPrice';

export default {
    title: 'Components/Card',
    component: CardPrice
} as Meta;

const Template: Story<CardPriceProps> = (args) => <CardPrice {...args} />

export const Price = Template.bind({});
Price.args = {
    description: 'Description',
    itemType: 'Item type',
    quantity: 2,
    price: 99900
}