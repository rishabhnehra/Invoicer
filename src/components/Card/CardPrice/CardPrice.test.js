import renderer from 'react-test-renderer';
import { CardPrice } from './CardPrice';

describe("<CardPrice />", () => {
    it("snapshot check", () => {
        const tree = renderer.create(<CardPrice description='Description' itemType='Item type' quantity={2} price={99900} />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});