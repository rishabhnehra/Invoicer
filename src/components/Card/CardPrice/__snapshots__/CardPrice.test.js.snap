// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardPrice /> snapshot check 1`] = `
<div
  className="CardPrice"
>
  <div
    className="headerSection"
  >
    <div
      className="description"
    >
      <h1>
        Description
      </h1>
      <p>
        Item type
      </p>
    </div>
    <div
      className="actions"
    >
      <img
        src="delete-black-18dp.svg"
      />
      <img
        src="create-black-18dp.svg"
      />
    </div>
  </div>
  <div
    className="priceSection"
  >
    <p
      className="quantity"
    >
      Qty: 
      2
    </p>
    <p
      className="price"
    >
      â‚¹  
      99900
    </p>
  </div>
</div>
`;
